core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -8716,10 +8716,22 @@ private void rollbackRemoveIndexFiles(CharSequence columnName, int columnIndex)
         }
     }
 
-    private void rollbackSymbolTables() {
+    private void rollbackSymbolTables(boolean quiet) {
         int expectedMapWriters = txWriter.unsafeReadSymbolColumnCount();
         for (int i = 0; i < expectedMapWriters; i++) {
-            denseSymbolMapWriters.getQuick(i).rollback(txWriter.unsafeReadSymbolWriterIndexOffset(i));
+            try {
+                denseSymbolMapWriters.get(i).rollback(txWriter.unsafeReadSymbolWriterIndexOffset(i));
+            } catch (CairoException e) {
```

Comment:
nit: there is no harm catching throwable instead, to avoid oopsies

Reply:
done in [203603a](https://github.com/questdb/questdb/pull/5598/commits/203603aa7c248ccd9e4dcc322b7c31f8f642684d)